modules/reitit-frontend/src/reitit/frontend/history.cljs

```diff
@@ -59,6 +59,14 @@
           el
           (recur (.-parentNode el)))))))
 
+(defn- event-target [event]
+  ;; Read event's target from composed path to get shadow dom working,
+  ;; fallback to target property if not available
```

Comment:
This would be better in a doc-string instead of a comment

Reply:
Thanks for the input. I changed this to doc-string.